### ISSUE TITLE
Will _C.so be open source?

### ISSUE THREAD

**Issue by qpc001** [nv-tlabs/NKSR#16]

**Comment by fwilliams** [comment]
Yes, it will (as part of something very exciting even beyond NKSR) but I can't give more details at the moment. Don't expect this until at least early next year though! 

Are there parts that you're particularly interested in?

**Comment by qpc001** [comment]
> Yes, it will (as part of something very exciting even beyond NKSR) but I can't give more details at the moment. Don't expect this until at least early next year though!
> 
> Are there parts that you're particularly interested in?

Thanks. That's enough.

**Comment by Saixiaoma** [comment]
@fwilliams Hi，When will _C.so be open source?

**Comment by Kin-Zhang** [comment]
@fwilliams also wait.... 

Think this two issues should be linked together: https://github.com/nv-tlabs/NKSR/issues/28

**Comment by ErenBalatkan** [comment]
> Yes, it will (as part of something very exciting even beyond NKSR) but I can't give more details at the moment. Don't expect this until at least early next year though!
> 
> Are there parts that you're particularly interested in?

Greetings, is there any update on this? Is the release delayed or cancelled?

**Comment by fwilliams** [comment]
Hi, @Saixiaoma @Kin-Zhang @ErenBalatkan  thanks a lot of following up on this.

We're releasing the DL framework for sparse 3D data that was used to implement _C.so at SIGGRAPH. Stay tuned! In the meantime, feel free to email me with specific questions. I can help offline.

**Comment by hungdche** [comment]
Hi @fwilliams is there a follow up to this? I'm particularly interested in CudaIndexGrid and its interface, and whether if I can build an oriented grid instead of axis-aligned. Thanks!

Edit: I guess it is just released on ArXiv (𝑓 VDB: A Deep-Learning Framework for Sparse, Large-Scale, and
High-Performance Spatial Intelligence).
